libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/internal/ManeuverComponent.kt

```diff
@@ -42,6 +43,11 @@ class ManeuverComponent(
                 contract?.get()?.onManeuverViewStateChanged(it)
             }
         }
+        coroutineScope.launch {
+            maneuverView.heightFlow.collect {
+                contract?.get()?.onManeuverViewHeightChanged(it)
+            }
+        }
```

Comment:
I guess we could simply track whether `ManeuverComponent` is currently attached, i.e. emit `true` in `onAttached` and `false` in `onDetached`, because you never use the actual height, you only compare it with 0. When maneuver view is hidden, `ManeuverComponent` is detached.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6381